frontend/javascripts/libs/mjs.js

```diff
@@ -209,6 +209,10 @@ V3.divide3 = function divide3(a, k, r) {
   return r;
 };
 
+V3.fromMag1ToMag = (vec, targetMag) => V3.floor(V3.divide3(vec, targetMag));
+
+V3.fromMagToMag1 = (vec, sourceMag) => V3.floor(V3.scale3(vec, sourceMag));
```

Comment:
I found it appealing to put it here, since the V3 module contains all operations which we perform on Vector3's. I know, that we haven't really used the domain knowledge of mags/resolutions here, but I think, it makes sense to do so. Overall, the entire mag/resolution handling (starting with the naming) could use a good cup of refactoring, obviously.. :see_no_evil: